docs/src/pages/components/trap-focus/trap-focus.md

```diff
@@ -0,0 +1,52 @@
+---
+title: Trap Focus React component
+components: Unstable_TrapFocus
+---
+
+# Trap Focus
+
+<p class="description">Trap focus within a DOM node.</p>
+
+> ⚠️ The component is experimental and unstable.
+
+`TrapFocus` is a utility component that manages focus for its descendants.
```

Comment:
Is there any function in calling it a "utility component"? It's a component. I think that's sufficient.